plugin/trino-mongodb/src/main/java/io/trino/plugin/mongodb/MongoMetadata.java

```diff
@@ -170,7 +172,7 @@ public void dropTable(ConnectorSession session, ConnectorTableHandle tableHandle
     @Override
     public void renameTable(ConnectorSession session, ConnectorTableHandle tableHandle, SchemaTableName newTableName)
     {
-        throw new UnsupportedOperationException();
+        throw new TrinoException(NOT_SUPPORTED, "This connector does not support renaming tables");
```

Comment:
remove the override, instead of copying wording from super

Reply:
Done! I got a bit lost when checking so many connectors :-)